flow/project.py

```diff
@@ -265,8 +277,135 @@ def make_bundles(operations, size=None):
     return _make_bundles(operations, size)
 
 
+class aggregate(object):
+    """Decorator for operation functions that needs to be aggregated.
+
+    .. code-block:: python
+        example_aggregate = aggregate()
+
+        @example_aggregate
+        @FlowProject.operation
+        def foo(jobs):
+            return len(jobs)
+
+    :param grouper:
+        Information on how to aggregate jobs.
+    :type grouper:
+        callable
+    :param sort:
+        Sort jobs by a statepoint parameter.
```

Comment:
When does this sort? Before or after grouping? These are the questions that people will come to the documentation with.